### ISSUE TITLE
Creating Pending/Onhold Status in version 1.9.3

### ISSUE THREAD

**Issue by mdmpofu** [osTicket/osTicket-1.8#2553]
Good people. 
i realise that this issue has been raised before but to osticket versions earlier than version 1.9. l am using version 1.9.3 and have tried to follow the mod customization of version 1.6 with no luck. anyone who has had a chance to customize the status to add a third status state besides open and closed like pending or onhold.

**Comment by ntozier** [comment]
All you have to do us upgrade?  Newer versions have custom status capability.

**Comment by mdmpofu** [comment]
Newer version have custom capability to create a new status but limits the state to open or closed. l want to have a scenario where l have a filter for Pending or Onhold tickets.

**Comment by ntozier** [comment]
You are correct that custom statuses are essentially sub types of open or closed... however you can use Advanced Search to search for tickets with custom statuses.

**Comment by shankarnewton** [comment]
i guess its worth mentioning that a new status with "Pending" should ideally have the clock stopped without ticking the clock.
So in-cases where you are awaiting response from a client and the delay is from the client side, your tickets are not breached and the reason is clear that the response from the client is still pending.
I checked this feature on new version of OST but still couldnt get it. 
let me know if this has been added to OST
